jcli/src/jcli_lib/vote/mod.rs

```diff
@@ -61,8 +61,8 @@ pub enum Error {
 pub enum Vote {
     /// Create committee member keys
     Committee(committee::Committee),
-    /// Build an encryption key from committee member keys
-    EncryptingKey(encrypting_vote_key::EncryptingVoteKey),
```

Comment:
Please change the example in docs that uses `--encrypting-key`.
Some other tooling may need to be updated as well; Cc @dkijania @danielSanchezQ